orttraining/orttraining/training_ops/cpu/nn/layer_norm.cc

```diff
@@ -36,12 +36,13 @@ LayerNormGrad<T>::LayerNormGrad(const OpKernelInfo& op_kernel_info)
 template <typename T>
 Status LayerNormGrad<T>::Compute(OpKernelContext* op_kernel_context) const {
   const Tensor* Y_grad = op_kernel_context->Input<Tensor>(0);
-  const Tensor* X = op_kernel_context->Input<Tensor>(1);
+  const Tensor* Y = op_kernel_context->Input<Tensor>(1);
   const Tensor* scale = op_kernel_context->Input<Tensor>(2);
-  const Tensor* mean = op_kernel_context->Input<Tensor>(3);
+  const Tensor* bias = op_kernel_context->Input<Tensor>(3);
   const Tensor* inv_std_var = op_kernel_context->Input<Tensor>(4);
 
-  const auto& X_shape = X->Shape();
+  const auto& Y_shape = Y_grad->Shape();
+  const auto& X_shape = Y_shape;
```

Comment:
Is this not the same thing?

Reply:
In the new op schema, there is no X as input.